app/services/edition_presenter_factory.rb

```diff
@@ -0,0 +1,16 @@
+module EditionPresenterFactory
```

Comment:
It's maybe a pattern associated with a more functional style of Ruby programming. We decided that these factory functions could work without any side-effects or dependencies.

No harm in changing it to be a `Class` with `self.methods` if that;'s the house style.